src/initializer.ts

```diff
@@ -6,10 +6,10 @@ import { EventBus, ExternalCallHandler, ExternalStorageHandler, ILiveConnect, Li
 import { EVENT_BUS_NAMESPACE } from './utils/consts'
 import { GlobalEventBus } from './events/event-bus'
 
-export function LiveConnect (liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, mode: string, externalEventBus?: EventBus): ILiveConnect {
+export function LiveConnect (liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, mode: string, externalEventBus?: EventBus): ILiveConnect | null {
```

Comment:
Why is the bus optional?

Reply:
We are creating our own when it's not passed in https://github.com/LiveIntent/live-connect/pull/103/files/bfcffb59ee3111e0f9d47c6e4b564b0b3ceed0ad#diff-16aa336504940f7a9c705777cc8e89acd054c3e54f132f6af961ada25aa6165dL11

Reply:
Ah yes.